packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
```

Comment:
1. Please don't disable eslint. There is a reason why these rules are set.
2. Please avoid let, use const. More info see [here](https://docs.scandipwa.com/structure/code-style/javascript-code-style#avoid-let-use-const).